index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="id-main-container">
+      <header><!--header of the document-->
+        <h1 id='title'> “Football Survey Form”</h1>
+        <p id='description'>Complete your registration</p>
+      </header>
+
+      <form id="survey-form">
+
+        <div id="id-getUserInfos"><!--gets the user infos and let him chose one player from the dropdown-->
+          <!--fieldset to enter information-->
+          <label id="name-label">Name</label>
+          <input id="name" type="text" required placeholder="input your name">
+
+          <label id="email-label">Email</label>
+          <input id="email" type="email" required placeholder="input your email">
+
+          <label id="number-label">Age</label>
+          <input id="number" type="number" required max="70" min="18" placeholder="age">
+
+          <select id="dropdown"><!--dropdown buttons-->
+            <option selected>Select your favorite player</option>
+            <option>Neymar</option> 
+            <option>Messi</option> 
+            <option>Cristiano Ronaldo</option> 
+          </select>
+        </div>
+
+        <div id="id-select-one-team"><!--let the user select one team with the radio button-->
+            <h3>What is your favorite team? (select one!)</h3>
+            <!--radio buttons-->
+            <input type="radio" id="id-psg-radio" value="PSG Team">
```

Comment:
```suggestion
            <input type="radio" id="id-psg-radio" value="PSG Team">
            <input type="radio" id="id-barcelona-radio" name="Team" value="Barcelona">
            <input type="radio" id="id-real-madrid-radio" name="Team" value="Real Madrid">
```
I think it would be better if you:
-make only one radio button clickable.
   + when you select one item the first one stays checked. To fix that, you can make all of them unchecked by default!

Reply:
you've done well using Inline comments, but you should add at least 2, also please re-add the summary with it too so the review looks complete

Reply:
well, thank you!